rgit.rb

```diff
@@ -122,11 +151,121 @@ def add(*pathspecs)
   def init(*arg)
     set_dir_info
     # If no dir is specified, use current dir, otherwise absolute path of specified dir
-    directory = arg.empty? ? @current_dir : File.absolute_path(arg[0])
+    directory = arg.empty? ? @current_dir : Pathname.new(arg[0]).realpath
     # thor hands us a frozen hash, ruby-git messes with it, so we hand it a shallow copy of the hash
     Repo.init(directory, options.dup)
     puts("Initialized empty rGit repository in #{directory}") unless options[:quiet]
   end
+ 
+
+  @clone_descriptions = {
+    local:
+      'When the repository to clone from is on a local machine, this flag
+       bypasses the normal "Git aware" transport mechanism and clones the
+       repository by making a copy of HEAD and everything under objects and
+       refs directories. The files under .git/objects/ directory are hardlinked
+       to save space when possible.
+
+       If the repository is specified as a local path (e.g., /path/to/repo),
+       this is the default, and --local is essentially a no-op. If the repository
+       is specified as a URL, then this flag is ignored (and we never use the local
+       optimizations). Specifying --no-local will override the default when
+       /path/to/repo is given, using the regular Git transport instead.',
+    bare:
+      ''
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.